docs/service-mesh.md

```diff
@@ -1,14 +1,18 @@
-# Deploying to an Istio-enabled cluster 
+# Deploying to an Istio-enabled cluster
 
-This repository provides an [`istio-manifests`](/istio-manifests) directory containing ingress resources (an Istio `Gateway` and `VirtualService`) needed to expose the app frontend running inside a Kubernetes cluster.
+This repository provides an [`istio-manifests`](/istio-manifests) directory
+containing ingress resources (an Istio `Gateway` and `VirtualService`) needed to
+expose the app frontend running inside a Kubernetes cluster.
```

Comment:
I haven't used Istio during the time I did the changes. 
The project is able to run with or without it.
Question is, do we wanna keep it?
No OTel is configured on it at the moment.